### ISSUE TITLE
Date and Time picker issue with voice-over

### ISSUE THREAD

**Issue by GlenStautland** [acidb/mobiscroll#502]
The time picker is not accessible with voice-over.  When "swiping" inside the calendar, the scroller for choosing time is not possible to get to without clicking directly in the time scroller. If the user clicks the time scroller, it's not possible to change the time

Version 5.15.2
